doc/recipe/preprocessor.rst

```diff
@@ -180,22 +180,68 @@ To get an overview on data fixes and how to implement new ones, please go to
 
 Fx variables as cell measures or ancillary variables
 ====================================================
-The following preprocessor may require the use of ``fx_variables`` 
-to be able to perform the computations:
+The following preprocessor may require the use of ``fx_variables`` to be able
+to perform the computations:
 
-    - ``area_statistics``
-    - ``mask_landsea``
-    - ``mask_landseaice``
-    - ``volume_statistics``
-    - ``weighting_landsea_fraction``
+    - area_statistics_ (``areacella``, ``areacello``)
+    - :ref:`mask_landsea<land/sea/ice masking>` (``sftlf``, ``sftof``)
+    - :ref:`mask_landseaice<ice masking>` (``sftgif``)
+    - volume_statistics_ (``volcello``)
+    - :ref:`weighting_landsea_fraction<land/sea fraction weighting>` (``sftlf``, ``sftof``)
```

Comment:
Perhaps format as table with columns preprocessor, applicable fx variables?
Also, why the use of `:ref:` only for some of the preprocessor?

Reply:
:ref: is only necessary for those that are not section titles yet. If you have a look at the compiled doc, you don't see any difference, all preprocessors are properly linked.

Reply:
You may want to read https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations. To quote:
> Using ref is advised over standard reStructuredText links to sections (like `Section title`_) because it works across files, when section headings are changed, will raise warnings if incorrect, and works for all builders that support cross-references.